src/Microsoft.SqlServer.Server/Microsoft.SqlServer.Server.csproj

```diff
@@ -40,9 +40,6 @@
       <LastGenOutput>Strings.Designer.cs</LastGenOutput>
     </EmbeddedResource>
   </ItemGroup>
-  <ItemGroup>
```

Comment:
For some reason even when the MicrosoftSourceLinkGitHubVersion package reference is defined in Versions.props, if it is removed from netcore project file the project will not build. It was removed from AKV project file without any issue.

Reply:
All I can guess is the MicrosoftSourceLinkGitHubVersion package reference is required to be defined inside the csproj for SDK project type.

Reply:
In light of the pipeline error message about not able to build test projects because of missing reference to Microsoft.SourceLink.GitHub package, the reference to Microsoft.SourceLink.GitHub was added back to each project so they will build correctly.